README.md

```diff
@@ -1,7 +1,11 @@
-# Base template for Gulp
+# All is Timed
 
 ## Task
 
+1. Using `setInterval`, create a page that would display the number of seconds the user spent on it. It will start by displaying `0`, after 1 second it will replace `0` with `1`, after another second it will show `2` and so on.
```

Comment:
```suggestion
1. Using `setInterval`, create a page that displays the number of seconds the user spent on it. Page loads with displayed `0`, after 1 second it will replace `0` with `1`, after another second it will show `2` and so on.
```